internal/guest/storage/scsi/scsi.go

```diff
@@ -217,8 +284,7 @@ func ControllerLunToName(ctx context.Context, controller, lun uint8) (_ string,
 		trace.Int64Attribute("controller", int64(controller)),
 		trace.Int64Attribute("lun", int64(lun)))
 
-	scsiID := fmt.Sprintf("0:0:%d:%d", controller, lun)
-
+	scsiID := fmt.Sprintf("%d:0:0:%d", controller, lun)
```

Comment:
Was this just inserting the controller at the wrong position before, and it only worked because it was always 0?

Reply:
Yep, this was a bug. But it always worked since controller was always 0.